storage/transactional/object.go

```diff
@@ -0,0 +1,63 @@
+package transactional
```

Comment:
No `Commit` method here, when the objects from temporal are copied to the main storer?